### DISCUSSION TITLE
does trace_io_uring_queue_async_work should add user_data parameter ?

### DISCUSSION THREAD

**Discussion by rowe0x** [axboe/liburing#1222] [Q&A] [answered]
The trace_io_uring_queue_async_work function should include the user_data field to correlate the trace data with submit and complete traces. This ensures that a request can be tracked through its entire lifecycle, from submission, through async work processing, to completion.

**Comment by axboe** [comment, marked as answer]
It does include user_data, maybe you are on an ancient kernel where it did not?

**Reply by rowe0x** [reply]
https://github.com/torvalds/linux/blob/v6.6/include/trace/events/io_uring.h#L149 
the kernel version I used is v5.10, I found v6.6 do include，thanks
